Sources/PayPalDataCollector/MagnesSDKProtocol.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import PPRiskMagnes
+
+protocol MagnesSDKProtocol {
+    func setUpWithParams(_ magnesParams: MagnesSetupParams) throws
+    func collectAndSubmit(
+        withPayPalClientMetadataId cmid: String,
+        withAdditionalData additionalData: [String: String]
+    ) throws -> MagnesResult
+}
+
+extension MagnesSDK: MagnesSDKProtocol {
+
+    func setUpWithParams(_ params: MagnesSetupParams) throws {
+        // workaround to avoid lint errors
+        try setUp(
+            setEnviroment: params.env,
```

Comment:
I know this is a magnes func, but we should probably tell them `environment` is spelled wrong. 😆

Reply:
Yeah def good thing it's internal haha.